docs/src/pages/getting-started/supported-components.md

```diff
@@ -32,7 +32,7 @@ to discuss the approach before submitting a PR.
 - **[Buttons: Floating Action Button](https://material.io/guidelines/components/buttons-floating-action-button.html) ✓**
   - **[Trigger](https://material.io/guidelines/components/buttons-floating-action-button.html#buttons-floating-action-button-transitions) ✓**
   - [Toolbar](https://material.io/guidelines/components/buttons-floating-action-button.html#buttons-floating-action-button-transitions)
-  - [Speed dial](https://material.io/guidelines/components/buttons-floating-action-button.html#buttons-floating-action-button-transitions)
+  - **[Speed dial](https://material.io/guidelines/components/buttons-floating-action-button.html#buttons-floating-action-button-transitions) ✓**
```

Comment:
Is a component in the lab makes it supported?
It doesn't even mean it will reach core. We might decide to kill one lab component for some reason.

Reply:
Legacy